src/vs/workbench/contrib/debug/browser/breakpointsView.ts

```diff
@@ -236,8 +236,14 @@ export class BreakpointsView extends ViewPane {
 		if (this.isBodyVisible()) {
 			this.updateSize();
 			if (this.list) {
+				const lastFocusIndex = this.list.getFocus()[0];
```

Comment:
`lastFocusIndex` can be `undefined`, since `getFocus` can return an empty array.
Also do not check against equality with -1.